pkg/scheduler/plugins/nodenumaresource/plugin.go

```diff
@@ -52,11 +52,14 @@ const (
 )
 
 var (
-	GetResourceSpec   = extension.GetResourceSpec
-	GetResourceStatus = extension.GetResourceStatus
-	SetResourceStatus = extension.SetResourceStatus
-	GetPodQoSClass    = extension.GetPodQoSClass
-	GetPriorityClass  = extension.GetPriorityClass
+	GetResourceSpec       = extension.GetResourceSpec
+	GetResourceStatus     = extension.GetResourceStatus
+	SetResourceStatus     = extension.SetResourceStatus
+	GetPodQoSClass        = extension.GetPodQoSClass
+	GetPriorityClass      = extension.GetPriorityClass
+	AllowByQoSAndPriority = func(qosClass extension.QoSClass, priorityClass extension.PriorityClass) bool {
```

Comment:
The name should try to hide the specific implementation inside. The purpose of this abstraction is to expect to be able to customize whether the implementation allows the use of CPUSet mode. AllowUseCPUSet might be more appropriate